src/og/input/KeyboardHandler.ts

```diff
@@ -1,25 +1,41 @@
 'use strict';
 
-import { input } from './input.js';
-import { stamp } from "../utils/shared.js";
+import {input} from './input';
+import {stamp} from "../utils/shared";
 
 const STAMP_SPACER = "_";
 
 class KeyboardHandler {
 
-    constructor() {
-        let _currentlyPressedKeys = {};
-        let _pressedKeysCallbacks = {};
-        let _unpressedKeysCallbacks = {};
-        let _charkeysCallbacks = {};
+    private _instance: any;
+
+    protected _removeCallback: any;
+    protected _getStamp: any;
+    protected _stamp: any;
+
+    public getcurrentlyPressedKeys: any;
```

Comment:
here I wanna cry( 
Try to descrybe all object instances as Record<some, some>

Reply:
No need to be so emotional. You could cry in another place:)